lib/tether/ops_linux.go

```diff
@@ -778,7 +778,7 @@ func (t *BaseOperations) MountLabel(ctx context.Context, label, target string) e
 	fi, err := os.Stat(target)
 	if err != nil {
 		// #nosec
-		if err := os.MkdirAll(target, 0744); err != nil {
+		if err := os.MkdirAll(target, 0755); err != nil {
```

Comment:
if we are changing all of these to `0755` could you make a constant for that value.

Reply:
I thought about that. I am not sure it is a constant type of value. It is a specific permission that I better see what it is instead of decoding a constant name.